src/less/components/panel.less

```diff
@@ -46,7 +46,7 @@
         }
     }
 
-    &.panel-no-padding {
+    &.panel-no-padding { //Is this used anywhere? [AW]
```

Comment:
Not sure, probably a question for a daily.. Extra points for signing the comment 👍